### DISCUSSION TITLE
Confusion on Multiplication with 10000000000

### DISCUSSION THREAD

**Discussion by jasjyotsinghjaswal** [smartcontractkit/full-blockchain-solidity-course-py#132] [Q&A] [answered]
Hi Patrick,
I understand that there is a limitation with decimals in solidity but I am not sure why in **getPrice** function we are multiplying with 10000000000 if we are eventually dividing with 10000000000 in **getConversionRate** . Wont avoiding multiplying in **getPrice** automatically avoid need of dividing by 10000000000 in **getConversionRate** . 
Just asking out of curiosity?

**Comment by cromewar** [comment, marked as answer]
Yeah you're right, however consider getPrice and GetConversionRate are independent functions,  and it could be a scenario where you use just getPrice and not getConversionRate so that's why is important to set the numbers as that.

**Reply by PatrickAlphaC** [reply]
Boom!
